### DISCUSSION TITLE
[Script request] Netbox

### DISCUSSION THREAD

**Discussion by Aloe-recite** [community-scripts/ProxmoxVE#313] [Request script] [answered]
### Application Name

Netbox

### Website

https://github.com/netbox-community/netbox

### Description

NetBox exists to empower network engineers. Since its release in 2016, it has become the go-to solution for modeling and documenting network infrastructure for thousands of organizations worldwide. As a successor to legacy IPAM and DCIM applications, NetBox provides a cohesive, extensive, and accessible data model for all things networked. By providing a single robust user interface and programmable APIs for everything from cable maps to device configurations, NetBox serves as the central source of truth for the modern network.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
Already as PR:
https://github.com/community-scripts/ProxmoxVE/pull/308
